python/ray/autoscaler/commands.py

```diff
@@ -344,15 +344,47 @@ def remaining_nodes():
 
             return head + workers
 
+        def try_docker_stop(node):
+            container_name = config.get("docker", {}).get("container_name")
```

Comment:
Is this the right way to check?
Shouldn't it be:
`if docker_config and docker_config["container_name"] != ""`?
container_name might be set to "" instead of None.

Reply:
Oh, let me switch to `if not container_Name`. 
`""` in Python evaluates to False & the second arg to `get()` is the 'default value'

Reply:
I understand, my main concern was that None and "" should return (not just None).